packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/Toolbar.tsx

```diff
@@ -90,6 +107,7 @@ const ToolbarButton = ({ icon, name, label, isActive, disabled, handleClick }) =
         asChild
       >
         <FlexButton
+          as="button" // needed for typescript
```

Comment:
Can you write why it's needed for typescript? Also you can probably remove it from the `attrs` function above?

Reply:
Having as set somehow is mandatory, and typescript isn't aware that the `attrs` function sets it for us. The easiest fix is to remove the attrs thing 👍

Reply:
yeah i think if you remove that and leave this there then it makes more sense 😄